subscripts/script_4_sol.m

```diff
@@ -0,0 +1,24 @@
+% This script has the fixed code if you are struggling to debug!
+
+%% SOLUTION: Breakout 1 
+%This section has the fixed code if you are struggling to debug!
+
+%read in data
+data = csvread('ASB.csv');
+%plot data
+plot(data(:,1), data(:,2), 'k-', 'LineWidth',7); %switched axes
+
+%% SOLUTION: Breakout 2
+
+%read in data
+data = csvread('ASB.csv');
+%plot data
+plot(data(:,1), data(:,2), 'k-', 'LineWidth',4); % change linewidth to 4
+
+%% SOLUTION: Breakout 2
+
+%read in data
+data = csvread('ASB.csv');
+%plot data
+plot(data(:,1), data(:,2), 'k-', 'LineWidth',4); % change linewidth to 4
+
```

Comment:
1. says "breakout 2" when it should read "breakout 3"
2. comments on line 23 not relevant to breakout 3.